server/plugins/webapp/index.js

```diff
@@ -9,6 +9,14 @@ const assert = require("assert");
 const HTTP_ERROR_500 = 500;
 const HTTP_REDIRECT = 302;
 
+const CONTENT_MARKER = "{{SSR_CONTENT}}";
+const HEADER_BUNDLE_MARKER = "{{WEBAPP_HEADER_BUNDLES}}";
+const BODY_BUNDLE_MARKER = "{{WEBAPP_BODY_BUNDLES}}";
+const TITLE_MARKER = "{{PAGE_TITLE}}";
+const PREFETCH_MARKER = "{{PREFETCH_BUNDLES}}";
+const META_TAGS_MARKER = "{{META_TAGS}}";
+const CRITICAL_CSS_MARKER = "{{CRITICAL_CSS}}";
```

Comment:
Moved these out of makeRouteHandler since we enforce a max statement rule, and there's no reason these need to be defined in makeRouteHandler